examples/official-site/robots.txt

```diff
@@ -0,0 +1,8 @@
+User-agent: *
+Disallow: /examples/authentification/  # Block the folder "authentification" to avoid blocking by password
```

Comment:
Maybe do not block the entire authentication example ? We still want it referenced. Ideally change the auth example so the password protected page has a different url, that you can block